kubernetes/product-deployment/scripts/testgrid-rds-cf.yaml

```diff
@@ -0,0 +1,48 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Parameters:
+  pDbUser:
+    Type: String
+  pDbPass:
+    Type: String
+  pDbEngine:
+    Type: String
+  pDbVersion:
+    Type: String
+  pDbInstanceClass:
```

Comment:
we can have a set of allowed values refer:- https://github.com/wso2-support/aws-apim/blob/4.0.0-u2/apim/Minimum-HA/apim.yaml#L1269 . This will help to have the best DB instance class for the deployment with regard to cost and performance.

Reply:
Added allowed values to pDbInstanceClass.